### PR TITLE
Use WITH_ZSTD=bundled

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build-ps/build-binary.sh [hunk 1]
```diff
@@ -258,6 +258,7 @@ fi
         -DDOWNLOAD_BOOST=1 \
 	-DFORCE_INSOURCE_BUILD=1 \
 	-DWITH_LIBEVENT=bundled \
+	-DWITH_ZSTD=bundled \
```
Comment (dutow) [inline build-ps/build-binary.sh]: Indentation of this line and the previous 2 seems off

**Review by vorsel — APPROVED** [review]

File: build-ps/percona-server.spec [hunk 2]
```diff
@@ -550,6 +551,7 @@ mkdir release
            -DWITH_ICU=bundled \
            -DWITH_READLINE=system \
            -DWITH_LIBEVENT=bundled \
+	   -DWITH_ZSTD=bundled \
```
Comment (dutow) [inline build-ps/percona-server.spec]: indentation

Comment (EvgeniyPatlan) [reply build-ps/percona-server.spec]: fixed

File: build-ps/build-binary.sh (hunk 1 above)
Comment (EvgeniyPatlan) [reply build-ps/build-binary.sh]: fixed

**Review by dutow — APPROVED** [review]

**Review by vorsel — APPROVED** [review]

**Pull request merged**